toplevel/opttoploop.ml

```diff
@@ -333,7 +333,8 @@ let execute_phrase print_outcome ppf phr =
         | _ -> sstr, false
       in
       let (str, sg, names, newenv) = Typemod.type_toplevel_phrase oldenv sstr in
-      if !Clflags.dump_typedtree then Printtyped.implementation ppf str;
+      if !Clflags.dump_typedtree then
+        Printtyped.implementation (Misc.Log.formatter log) str;
```

Comment:
Note for myself: the calls to `Misc.Log.formatter` correspond to places that we probably want, later, to turn into more structured calls that would include the inputs in the JSON. It is fine to have them here during a transition period.

Reply:
Indeed, those calls should disappear in the part 2 of this PR.